dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -14,7 +14,7 @@ class WorkshopDailySurveyController < ApplicationController
     # for the relevant session id.
     # The pre-workshop survey, which has no session id, will redirect to thanks.
     def new_general
-      workshop = Workshop.where(subject: SUBJECT_SUMMER_WORKSHOP).enrolled_in_by(current_user).nearest
+      workshop = Workshop.where(subject: SUBJECT_TEACHER_CON).or(Workshop.where(subject: SUBJECT_SUMMER_WORKSHOP)).enrolled_in_by(current_user).nearest
```

Comment:
I'm not sure exactly how the order of operations works with `or` and the `enrolled_in_by` here; it would probably be better to just do `Workshop.where(subject: [SUBJECT_TEACHER_CON, SUBJECT_SUMMER_WORKSHOP])`

Reply:
They should be equivalent (this is newer syntax I believe) but I can do the other way if that is more readable.

http://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-or